gon/shaped.py

```diff
@@ -17,22 +18,25 @@
 from .linear import (Contour,
                      RawContour,
                      forms_convex_polygon,
-                     to_area)
+                     to_signed_area)
 from .primitive import Point
 
 RawPolygon = Tuple[RawContour, List[RawContour]]
 
 
 class Polygon(Geometry):
-    __slots__ = '_border', '_holes', '_raw_border', '_raw_holes'
+    __slots__ = ('_border', '_holes', '_raw_border', '_raw_holes',
+                 '_normalized_border', '_normalized_holes', '_is_normalized')
 
     def __init__(self, border: Contour,
-                 holes: Optional[Sequence[Contour]] = None) -> None:
+                 holes: Optional[Sequence[Contour]] = None,
+                 *,
+                 _is_normalized: bool = False) -> None:
```

Comment:
I have doubts about this. Probably we shouldn't allow a user to specify if the polygon is normalized or not.

Reply:
that's why it is underscored